src/test/java/com/mediamarktsaturn/technolinator/git/TechnolinatorConfigTest.java

```diff
@@ -47,7 +47,6 @@ void testFullConfig() throws JsonProcessingException {
                 location: projectLocation
                 recursive: false
             gradle:
-                multiProject: true
```

Comment:
Would it make sense to keep this line as is, as removing a config might be a braking change?

Reply:
good point, thanks.
the test uses a different objectmapper than the github app extension. 
will try to reuse the production one, or at least configure the test one the same.